src/renderers/WebGLRenderer.js

```diff
@@ -1147,6 +1151,61 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 	}
 
+	this.renderOverride = function ( overrideMaterial, scene, camera, renderTarget, clearColor, clearAlpha ) {
+
+		var originalClearColor = this.getClearColor(), originalClearAlpha = this.getClearAlpha(), originalAutoClear = this.autoClear;
+
+		this.autoClear = false;
```

Comment:
Do we consider the case when there's no far-clipping (like in a strictly indoor scene) and don't need to clear?